public/stylesheets/admin.css

```diff
@@ -1,5 +1,7 @@
 body {
   font-family: 'Amatic SC', sans-serif;
+  margin-top: 25px;
```

Comment:
Just to decrease the amount of "stuff" getting sent over the wire, whenever I have multiple margin or padding rules on the same element, I tend to collapse to a single rule (where it saves characters):
```css
  margin: 25px 0;
```